### PR TITLE
New version: PhapDieuDuong.ExifGlass version 1.4.1.0

### PR DESCRIPTION
**Pull request by Exorcism0666**
###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/133852)

### CODE REVIEW

**Comment by Exorcism0666** [comment]
## For moderators

This account is automated by Github Actions and the source code was created by CoolPlayLin. If you have any questions about any pull request, don't hesitate to ping @Exorcism0666, I'll get a notification.
>[!important]
>Please carefully review these Pull Request before merging. If it is a Pull Request for removing incorrect content and the URLs are issue, free upon manual checking, please close this Pull Request directly. (It is best to inform Exorcism0666 of the closure.)

良い一日をお過ごしください！

**Comment by stephengillie** [comment]
Automatic Validation ended with:
>  Installation failed with exit code -1978335141 MSI (s) (44:30) [19:50:46:324]: Executing op: ComponentRegister(ComponentId={3CC2DE97-A72C-5898-A8C6-2127ABFCAC71},KeyPath=22*85*\SOFTWARE\Microsoft\Windows\Windows Error Reporting\RuntimeExceptionHelperModules\C:\Program Files\dotnet\shared\Microsoft.NETCore.App\8.0.1\mscordaccore.dll,State=3,,Disk=1,SharedDllRefCount=0,BinaryType=1) MSI (s) (44:30) [19:50:47:699]: Executing op: RegOpenKey(Root=-2147483646,Key=SOFTWARE\Microsoft\Windows\Windows Error Reporting\RuntimeExceptionHelperModules,,BinaryType=1,,) Property(S): ErrorDialog = ErrorDlg MSI (s) (44:30) [19:50:47:964]: Windows Installer installed the product. Product Name: Microsoft .NET Runtime - 8.0.1 (x64). Product Version: 64.4.5793. Product Language: 1033. Manufacturer: Microsoft Corporation. Installation success or error status: 0. MSI (s) (44:BC) [19:50:48:558]: Windows Installer installed the product. Product Name: Microsoft .NET Host FX Resolver - 8.0.1 (x64). Product Version: 64.4.5793. Product Language: 1033. Manufacturer: Microsoft Corporation. Installation success or error status: 0. MSI (s) (44:FC) [19:50:49:824]: Windows Installer installed the product. Product Name: Microsoft .NET Host - 8.0.1 (x64). Product Version: 64.4.5793. Product Language: 1033. Manufacturer: Microsoft Corporation. Installation success or error status: 0. MSI (s) (44:68) [19:50:56:796]: Windows Installer installed the product. Product Name: Microsoft Windows Desktop Runtime - 8.0.1 (x64). Product Version: 64.4.5797. Product Language: 1033. Manufacturer: Microsoft Corporation. Installation success or error status: 0.

(Automated response - build 566.)

**Comment by stephengillie** [comment]
Manual Validation ended with:
> 2024-01-12 15:59:48.429 [CLI ] Extracting archive to: C:\Users\User\AppData\Local\Temp\WinGet\PhapDieuDuong.ExifGlass.1.4.1.0\extracted 2024-01-12 15:59:50.051 [CLI ] Successfully extracted archive 2024-01-12 15:59:50.053 [CLI ] Unable to locate nested installer at: C:\Users\User\AppData\Local\Temp\WinGet\PhapDieuDuong.ExifGlass.1.4.1.0\extracted\ExifGlass_1.4.1.0.0_net8_x64\ExifGlass.exe

(Automated response - build 566.)

**Comment by stephengillie** [comment]
Hi @Exorcism0666,

> This PR's version number 1.4.1.0 has 4 parameters (sets of numbers between dots - major, minor, etc), which is greater than the current manifest's version 1.4.1, which has 3 parameters.

Is this intentional?

(Automated response - build 566)

File: manifests/p/PhapDieuDuong/ExifGlass/1.4.1.0/PhapDieuDuong.ExifGlass.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Created with Komac v1.11.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: PhapDieuDuong.ExifGlass
+PackageVersion: 1.4.1.0
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: ExifGlass_1.4.1.0.0_net8_x64\ExifGlass.exe
```
Comment (stephengillie) [inline manifests/p/PhapDieuDuong/ExifGlass/1.4.1.0/PhapDieuDuong.ExifGlass.installer.yaml:9]:
```suggestion
- RelativeFilePath: ExifGlass_1.4.1.0_net8_x64\ExifGlass.exe
```

**Comment by Exorcism0666** [comment]
> Hi @Exorcism0666,
> 
> > This PR's version number 1.4.1.0 has 4 parameters (sets of numbers between dots - major, minor, etc), which is greater than the current manifest's version 1.4.1, which has 3 parameters.
> 
> Is this intentional?
> 
> (Automated response - build 566)

Yes!

**Review by stephengillie — APPROVED** [review]

**Pull request merged**